packages/kotti-ui/source/kotti-user-menu/types.ts

```diff
@@ -1,16 +1,27 @@
-import { KottiNavbar } from '../kotti-navbar/types'
+import { z } from 'zod'
+
 import { SpecifyRequiredProps } from '../types/utilities'
 
 export namespace KottiUserMenu {
-	export type SectionLink = KottiNavbar.SectionLink
-	export type Section = KottiNavbar.Section
+	export const sectionLinkSchema = z.object({
+		title: z.string(),
```

Comment:
according to the usage

there is `link.link` so add `link: z.string()`

Reply:
apparently there is a handler for external control of the click event, so it's `string | undefined`

Reply:
i.e. 
add `link: z.string().optional()`